frontend/app/components/atoms/textbutton/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import styles from "../../../../styles/TextButton.module.scss";
+import Dropdown from "../drop-down/index";
+
+interface ButtonProps {
+  label: string;              // Text to display inside the button
+  onClick: () => void;        // Function to call when button is clicked
```

Comment:
when the button is pressed, we just want to call this `onClick` function and nothing else